bin/update-draft-release

```diff
@@ -18,13 +18,17 @@ options = {}
 # --open-url
 #
 ARGV.each do |arg|
-  case arg
-  when '--at-the-end'
-    options[:insert_at_the_end] = true
-  when '--before-gamma'
-    options[:insert_before] = '## Gamma ##'
-  when '--open-url'
+  case
+  when arg.match(/^--in-[A-z-]+$/)
```

Comment:
specify `--in-top-level` outside as a special option :) so don't have to support `-` in regex match